forward_client.py

```diff
@@ -0,0 +1,38 @@
+# USAGE
+# python client.py --server-ip SERVER_IP
+
+# import the necessary packages
+from imutils.video import VideoStream
+import imagezmq
+import argparse
+import socket
+import time
+import cv2
+
+# construct the argument parser and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-s", "--server-ip", required=True,
+	help="ip address of the server to which the client will connect")
+ap.add_argument("-c", "--camera-ip", required=True, help="ip address of the camera on the local network to get a feed from")
```

Comment:
is this currently set up for forwarding on camera?

Reply:
currently set up where?

Reply:
like forward_client, does it just forward one camera?

Reply:
as of right now yes, can edit to send multiple. basically taking the structure or idea of our original code and implementing it in the forwarder

Reply:
cool, i have an idea for this so i'll slack u